yarn-project/world-state/src/native/native_world_state_instance.ts

```diff
@@ -257,6 +263,8 @@ export class NativeWorldState implements NativeWorldStateInstance {
       throw new Error('Invalid response message type: ' + response.msgType + ' != ' + messageType);
     }
 
+    this.instrumentation.recordRoundTrip(callDuration * 1000, messageType);
```

Comment:
I don't think this is going to give us the resolution we want. We should be using `process.hrtime` for this https://nodejs.org/api/process.html#processhrtimebigint